src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -101,23 +101,33 @@ public DebuggedProcess(bool bLaunched, LaunchOptions launchOptions, ISampleEngin
                         _waitDialog.ShowWaitDialog(file);
                     }
                 }
-                else if (results.Results.Contains("shlib-info")) //lldb
+                else if (this.MICommandFactory.Mode == MIMode.Clrdbg)
```

Comment:
NOTE: Now that the lldb library load event is more applicable, we should update clrdbg to match lldb. I added a cleanup item to the clrdbg list.